.github/workflows/test-unit.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     name: Unit
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/mvorisek/image-php:${{ matrix.php }}
+      image: ghcr.io/mvorisek/image-php:${{ matrix.php }}-debian
```

Comment:
Alpine support for newer pdo_oci is broken, see https://github.com/mlocati/docker-php-extension-installer/issues